keyboards/planck/keymaps/dudeofawesome/keymap.c

```diff
@@ -61,7 +61,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
   /* Workman
    * ,-----------------------------------------------------------------------------------.
-   * | Tab  |   Q  |   W  |   R  |   W  |   B  |   J  |   F  |   U  |   P  |   ;  | Bksp |
+   * | Tab  |   Q  |   D  |   R  |   W  |   B  |   J  |   F  |   U  |   P  |   ;  | Bksp |
```

Comment:
Is this your keymap?

Reply:
It's a fix for a typo in a comment.
The Workman keymap is `QDRW...`, the code is good but the comment is wrong.

That's the one I use in order to make my PR and it took me a while to see the typo that why I fit it.
Should I made another PR for dudeofawesome comments?